libraries/plugins/rc/include/steem/plugins/rc/rc_curve.hpp

```diff
@@ -0,0 +1,38 @@
+
+#pragma once
+#include <steem/plugins/rc/rc_utility.hpp>
+
+#include <steem/protocol/types.hpp>
+
+#include <fc/time.hpp>
+
+namespace steem { namespace plugins { namespace rc {
+
+struct rc_curve_gen_params
+{
+   uint16_t                inelasticity_threshold_num    = 1;
+   uint16_t                inelasticity_threshold_denom  = 128;
+   uint16_t                a_point_num                   = 1;
```

Comment:
These are not the values that we currently use but are the default script values.

```
"a_point_num" : 37,
"a_point_denom" : 1000,
"u_point_num" : 1,
"u_point_denom" : 2
```